docs/src/main/asciidoc/metrics-guide.adoc

```diff
@@ -0,0 +1,180 @@
+include::./attributes.adoc[]
+= {project-name} - MicroProfile Metrics
+
+This guide demonstrates how your {project-name} application can utilize the MicroProfile
+Metrics specification through the SmallRye Metrics extension.
+
+MicroProfile Metrics allows applications to gather various metrics and statistics that provide
+an insight into what is happening inside the application.
+
+The metrics can be read remotely using JSON format or the OpenMetrics format, so that
+they can be processed by additional tools such as Prometheus, and stored for analysis
+and visualisation.
+
+== Prerequisites
+
+To complete this guide, you need:
+
+* less than 15 minutes
+* an IDE
+* JDK 1.8+ installed with `JAVA_HOME` configured appropriately
+* Apache Maven 3.5.3+
+
+== Architecture
+In this example, we build a very simple microservice which offers one REST endpoint and that endpoint serves
+for determining whether a number is prime. The implementation class is annotated with some metric annotations
+so that while responding to user's requests, some metrics will be gathered. The meaning of each metric will be explained later.
```

Comment:
I think the present tense would be better: `some metrics are gathered`.

I think you can drop "The meaning of each metric will be explained later.". It doesn't bring anything.